docker/mysql/init.sql

```diff
@@ -9,7 +9,7 @@ CREATE TABLE metadata_aspect_v2 (
   createdby                     VARCHAR(255) NOT NULL,
   createdfor                    VARCHAR(255),
   CONSTRAINT pk_metadata_aspect_v2 PRIMARY KEY (urn,aspect,version)
```

Comment:
should we change our smoke test to insert some UTF8 metadata and make sure it comes back unharmed?

Reply:
good point. let me try adding